core/modules/parser/QSMySqlListener.cc

```diff
@@ -934,6 +980,19 @@ class FromClauseAdapter :
         _groupByClause->addTerm(query::GroupByTerm(valueExpr, ""));
     }
 
+    void checkContext() const override {
+        // required:
+        assertExecutionCondition(__FUNCTION__, _ctx->FROM() != nullptr, "Context check failure.", _ctx);
+        // optional:
```

Comment:
Multiple comments lines are just confusing. If they're just temporarily disabled then, perhaps this should be explained.